test/Testing.py

```diff
@@ -19,6 +20,11 @@ def testpath():
 IS_X86_64 = platform.machine() == 'x86_64'
 IS_I686 = re.match(platform.machine(), 'i[3456]86')
 HAS_32BIT_GLIBC = glob.glob('/lib/ld-linux.so.*')
+HAS_CHECKBASHISMS = shutil.which('checkbashisms')
+HAS_DASH = shutil.which('dash')
+HAS_DESKTOP_FILE_UTILS = shutil.which('desktop-file-validate')
+HAS_APPSTREAM_GLIB = shutil.which('appstream-util')
+HAS_RPMDB = Path('/var/lib/rpm').exists()
```

Comment:
This will make tests fail on openSUSE, since their rpmdb is not in `/var/lib/rpm`. The correct way to get the rpmdb path is to use `rpm --eval "%_dbpath"`.